packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1054,7 +1067,9 @@ class EditableText extends StatefulWidget {
   /// {@template flutter.widgets.editableText.inputFormatters}
   /// Optional input validation and formatting overrides.
   ///
-  /// Formatters are run in the provided order when the text input changes.
+  /// Formatters are run in the provided order when the text input changes. When
+  /// this parameter changes, the new formatters will not be applied until the
+  /// next time the user inserts or deletes text.
```

Comment:
Should `inputFormatters` work retroactively?

Reply:
Good question...  I'm not sure.  I guess it would be easy for the user to apply their input formatter to the existing text if they want to.  Maybe some users are doing that now.  I guess I don't have a strong reason to change it.